projects/ngx-favicon/src/lib/ngx-favicon.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface NgxFaviconConfig<CustomFavicon> {
+  targetId: string;
```

Comment:
I think this name is a little unclear, I'm guessing it is the element that the favicon is declared on? In that case I think it should be something like `faviconElementId`

Reply:
Updated